charts/sourcegraph-appliance/templates/hpa.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.autoscaling.enabled }}
```

Comment:
Low-confidence answer: the way we instantiate the controller manager, we _might_ get some fighting between concurrent reconcilers, because we don't enable the controller-runtime leader election configurations: https://book.kubebuilder.io/faq#after-make-run-i-see-errors-like-unable-to-find-leader-election-namespace-not-running-in-cluster

It still might be fine, because in theory concurrent controllers should converge to the same result, but they _might_ fight each other. We could enable the leader election stuff, and it should "just work", but it is complexity we might not need for a 1st iteration.

I wouldn't worry about the need for multiple instances to handle the web server traffic either, it should be vanishingly low on any given instance.

Reply:
Honestly, I don't think we need an HPA for appliance, and it would be silly to manage that. The amount of traffic we'd receive on each deployment cannot meaningfully exceed the capacity of a single replica. This was instantiated by default on `helm create` and I should've removed it.

Reply:
Ah, gotcha, I always forget about that helm-create scaffolding!